### ISSUE TITLE
Question on setting random seed

### ISSUE THREAD

**Issue by lizhenstat** [iiradia/kPOD#3]
Hi, thanks for your kPOD work
I have one question related with random initialization,
sklearn.cluster.KMeans has a parameter **random_state** for reproducing same clustering result
I was wondering if there is such features(parameters) in kPOD to reproduce clustering result.

Thanks

**Comment by lizhenstat** [comment]
Hi, I find the answer to this question, just add a random_state parameter before [this line](https://github.com/iiradia/kPOD/blob/master/kPOD/utils/initial_helpers.py#49)  and change the corresponding function's parameter will produce the same clustering result:

```python
    np.random.seed(random_state) # sirius added a seed to the random initialization
    centroids = []
    centroids.append(data[np.random.randint(N), :])
```
